learn/cookbooks/azure.md

```diff
@@ -0,0 +1,78 @@
+# Deploy a Meilisearch instance on Azure
+
+_The following is a guest post by Christopher Maneu, Data Engineering Lead Advocate at Microsoft._
+
+## Introduction
+
+In this tutorial, we will see how to deploy a Meilisearch instance on Azure, and also some considerations for hosting a production Meilisearch instance on Azure.
+
+::: tip
+If you don't have yet an Azure subscription, you can [create one here](https://azure.microsoft.com/free/). With the Azure Free tier, you can run a Meilisearch instance for free for at least 12 months.
+:::
+
+## One-Click Deploy
+
+You can deploy a Meilisearch instance via the official [Meilisearch Docker image](https://hub.docker.com/r/getmeili/meilisearch) by clicking the following button:
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fcmaneu%2Fmeilisearch-on-azure%2Fmain%2Fmain.json).
+
+![The Azure portal prompting for information to deploy Meilisearch](/azure/01.azure-deploy-button.png)
+
+When clicking this button, you'll be redirected to the Azure Portal, and asked a few questions:
+
+- **Region**: This is the deployment region. You should select a region close to you or your users,
+- **Environment**: The name of the environment you want to deploy to, like `dev` (for development) or `prod` (for production). This settings has no effects on the type of resources deployed. It's just for convenience.
+- **Application Name**: The name of your application. This name should be **unique across all Azure customers**. The URL where your application is deployed will contain this value.
+- **Location**: This the resources location. For reasons out of scope of this documentation, this parameter is different from the _Region_ one. By default, it is set to `eastus`, but you can type any Azure location (like `francecentral`, `westeurope`, or `japaneast`).
+- **Meilisearch_apikey**: This is the Meilisearch Master Key. While we generate a random key by default, we encourage you to set your own key.
```

Comment:
```suggestion
- **Meilisearch_apikey**: This will be the [master key](/learn/advanced/security.md) of your Meilisearch instance. While we generate a random key by default, we encourage you to set your own key.
```